specification/changeanalysis/resource-manager/Microsoft.ChangeAnalysis/preview/2020-04-01-preview/changeanalysis.json

```diff
@@ -0,0 +1,432 @@
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/ConfigurationProfileResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ConfigurationProfile"
+        ],
+        "summary": "Updates configuration profile for for the Microsoft.ChangeAnalysis resource provider.\r\nThis operation is expected to be idempotent. In current version of API name of the profile is ignored.",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json",
+          "default"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ProfileNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Request body containing new configuration profile.",
+            "schema": {
+              "$ref": "#/definitions/ConfigurationProfileResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/ConfigurationProfileResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "ConfigurationProfile"
+        ],
+        "summary": "Updates a configuration profile with the specified parameters.",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json",
+          "default"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ProfileNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The parameters for updating a configuration store.",
+            "schema": {
+              "$ref": "#/definitions/ConfigurationProfileResource"
```

Comment:
PATCH has the same request body as PUT? Will all mutable properties be supported for update with PATCH? PATCH is not mandatory for proxyOnly resources (but is good to have if the service supports)

Reply:
Yes the number of mutable properties is very limited - it's just notification configuration settings .